portal/server/generateApp.ts

```diff
@@ -61,7 +69,18 @@ const generateApp = () => {
     };
   }
 
-  const redisClient = redis.createClient(process.env.REDIS_PORT, process.env.REDIS_HOSTNAME, redisOptions);
+  const redisPort = process.env.REDIS_PORT as string;
+
+  // if (!process.env.REDIS_PORT || Number.isNaN(parseInt(process.env.REDIS_PORT, 10))) {
+  //   console.error('Portal UI server - invalid REDIS_PORT', process.env.REDIS_PORT);
+  //   throw new InvalidEnvironmentVariableError('Invalid redis port value.');
+  // }
```

Comment:
Is this commented out code supposed to be here?

Reply:
Had a comment on previous PR forgot to add

The REDIS_PORT env var is undefined but redis.createClient() doesn't allow this to be passed in.
Assume this variable is used in prod but not sure how to validate it